ResearchKit/Common/ORKFormStepViewController.m

```diff
@@ -782,7 +841,11 @@ - (ORKStepResult *)result {
     
     NSMutableArray *qResults = [NSMutableArray new];
     for (ORKFormItem *item in items) {
-
+        
+        if ([_filteredForms containsObject:item]) {
```

Comment:
What would you think about the name `_hiddenFormItems` instead of `_filteredForms`? imo "filter/filtered" is often ambiguous as a variable name.